rolling-shutter/keyperimpl/shutterservice/config.go

```diff
@@ -0,0 +1,153 @@
+package shutterservice
+
+import (
+	"io"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/shutter-network/rolling-shutter/rolling-shutter/keyper/kprconfig"
+	"github.com/shutter-network/rolling-shutter/rolling-shutter/medley/configuration"
+	"github.com/shutter-network/rolling-shutter/rolling-shutter/medley/metricsserver"
+	"github.com/shutter-network/rolling-shutter/rolling-shutter/p2p"
+)
+
```

Comment:
This is missing an assignment check to verify the structs actually satisfy the interface. See for example: https://github.com/shutter-network/rolling-shutter/blob/bae90acdf19f17168315b29d6f38f68f86f7f26c/rolling-shutter/keyperimpl/gnosis/config.go#L21-L25